devguide/howtos/add-function.md

```diff
@@ -63,8 +63,19 @@ The parameters passed to call are all optional (except ~name and ~in_product_sin
   permitted to merely be a single value rather than a list of values. If no
   ~result is specified, the default is unit.
 
+- The ~doc parameter describes what it is, the message is doing.
```

Comment:
Perhaps:
describes what the message is doing
describes the purpose of the message